main_backend.py

```diff
@@ -1,6 +1,11 @@
+from config import config_dict
+
+if "USE_DB" in config_dict.keys() and config_dict["USE_DB"]:
```

Comment:
This can be simplified slightly to be a bit more readable by using the `get` method of dictionaries in python.
```py
if config_dict.get("USE_DB", False):
    import print_db as print_queue  # use db print queue
else:
    import print_sheet as print_queue  # use sheet print queue
```

Reply:
Good to know, pretty sure this got removed with the final commit anyway

Reply:
Ah cool I think there might be some areas where this pattern is used so its handy to know of an alternative